### ISSUE TITLE
Kotlin support -lang kt and -lang kotlin

### ISSUE THREAD

**Issue by colleend** [semgrep/semgrep#2507]
**Describe the bug**
-lang kt and -lang kotlin from the semgrep repository currently result in 'language not found' errors.

**Comment by aryx** [comment]
what does "from the semgrep repository" means in this context?
What is the command-line that fails?

**Comment by SPoint42** [comment]
This 2 lines : 

`docker run -v ${PWD}:/src -v c:\semgrep-rules\Level1\Kotlin\:/rules  returntocorp/semgrep /src --config /rules/ --lang kt`
`docker run -v ${PWD}:/src -v c:\semgrep-rules\Level1\Kotlin\:/rules  returntocorp/semgrep /src --config /rules/ --lang kotlin`
returns : 

`unsupported language: kt. supported languages are: C#, JSON, Json, c, cs, csharp, generic, go, golang, java, javascript, js, json, jsx, lua, ml, none, ocaml, php, py, python, python2, python3, rb, regex, ruby, ts, tsx, typescript`

or `unsupported language: kotlin. supported languages are: C#, JSON, Json, c, cs, csharp, generic, go, golang, java, javascript, js, json, jsx, lua, ml, none, ocaml, php, py, python, python2, python3, rb, regex, ruby, ts, tsx, typescript`

with 


```rules:
  - id: kotlin-path-traversal
    patterns:
      - pattern-either:
          - pattern: |
              createTempDir($VAR1,$VAR2)
          - pattern: |
              createTempFile($VAR1,$VAR2)
          - pattern: |
              createTempDir($VAR1,$VAR2, File("..."))
          - pattern: |
              createTempFile($VAR1,$VAR2, File("..."))
    languages:
      - kt
    message:
      Untrusted user input in createTempDir()/createTempFile() can endup in Directory Traversal Attacks.
    severity: ERROR
    metadata:
      owasp: 'A5: Broken Access Control'
      cwe: 'CWE-23: Relative Path Traversal
```

or with
 
```rules:
  - id: kotlin-path-traversal
    patterns:
      - pattern-either:
          - pattern: |
              createTempDir($VAR1,$VAR2)
          - pattern: |
              createTempFile($VAR1,$VAR2)
          - pattern: |
              createTempDir($VAR1,$VAR2, File("..."))
          - pattern: |
              createTempFile($VAR1,$VAR2, File("..."))
    languages:
      - kotlin
    message:
      Untrusted user input in createTempDir()/createTempFile() can endup in Directory Traversal Attacks.
    severity: ERROR
    metadata:
      owasp: 'A5: Broken Access Control'
      cwe: 'CWE-23: Relative Path Traversal'
```

**Comment by aryx** [comment]
I think this should work on develop now.
Should be available in the next semgrep release (0.40.0) next week.
